nipype/interfaces/ants/resampling.py

```diff
@@ -246,9 +246,13 @@ class ApplyTransformsInputSpec(ANTSCommandInputSpec):
     interpolation_parameters = traits.Either(traits.Tuple(traits.Int()),  # BSpline (order)
                                              traits.Tuple(traits.Float(),  # Gaussian/MultiLabel (sigma, alpha)
                                                           traits.Float())
-                                             )
-    transforms = InputMultiPath(
-        File(exists=True), argstr='%s', mandatory=True, desc='transform files: will be applied in reverse order. For example, the last specified transform will be applied first')
+                                            )
+    transforms = traits.Either(InputMultiPath(File(exists=True), desc='transform files: will be '
```

Comment:
I wanted to be able to pass an identity transform (use case: ds005 in fmriprep). This way, one can pass either a list of transform files OR the keyword 'identity'. It makes the code much cleaner in fmriprep, where I can say "transform the epi files with transformation ___"  for ds005 AND ds054, so I don't have to special-case. In some ways it is a workaround for not having conditional workflows in nipype.